libraries/abstractions/pkcs11/test/iot_test_pkcs11.c

```diff
@@ -279,15 +281,37 @@ TEST_GROUP_RUNNER( Full_PKCS11_RSA )
 {
     #if ( pkcs11testRSA_KEY_SUPPORT == 1 )
         prvBeforeRunningTests();
+        #if ( pkcs11testIMPORT_PRIVATE_KEY_SUPPORT == 1 )
+            RUN_TEST_CASE( Full_PKCS11_RSA, AFQP_CreateObject );
+        #endif
+
+        /* Generating RSA keys is not currently supported.
```

Comment:
If not needed, can be removed?

Reply:
I'll have to think about this, it would be nice to keep the test code should this use case arise